index.html

```diff
@@ -0,0 +1,484 @@
+        <li><a class="link partner-link" href="#sponsor">Sponsor</a></li>
+      </ul>
+    </div>
+
+    <main class="main" id="home">
+      <nav class="nav">
+        <ul class="nav-top-list">
+          <li>
+            <a
+              href="https://www.facebook.com/mukhammadrizo.abdunazarov"
+              target="_blank"
+              rel="noopener noreferrer"
+              ><i class="fab fa-facebook fa-lg"></i>
+            </a>
+          </li>
+          <li>
+            <a
+              href="https://twitter.com/Mukhammadrizo5"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <i class="fab fa-twitter fa-lg"></i>
+            </a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li>
+            <a href="https://github.com/mukhammadrizooff/2022-G20-Bali-summit"
+              >My Page</a
+            >
+          </li>
+          <li>
+            <a href="https://github.com/mukhammadrizooff/2022-G20-Bali-summit"
+              >Logout</a
+            >
+          </li>
+        </ul>
+        <div class="desktop-menu">
+          <a class="logo" href="#home">
+            <img alt="logo" src="images/logo.png" height="90" width="180" />
+          </a>
+          <ul class="desktop-menu-list">
+            <li><a class="link about-link" href="#about">About</a></li>
+            <li><a class="link" href="#program-id">Program</a></li>
+            <li><a class="link" href="#speaker">Join</a></li>
+            <li><a class="link" href="#partner">Partner</a></li>
+            <li><a class="link" href="#">News</a></li>
+            <li class="menu-list-btn">
+              <a class="link" href="#home">Media Centre</a>
+            </li>
+          </ul>
+        </div>
+      </nav>
+
+      <header class="head">
+        <i class="fas fa-bars fa-3x wrap-btn open"></i>
+        <h1 class="head-top">"Welcome to Indonesia!"</h1>
+        <div class="h2-bg">
+          <h2 class="head-main">
+            2022 G20 <br />
+            Bali summit
+          </h2>
+        </div>
+        <p class="main-text">
+          In the context of the G20, this illustrates the determination of the
+          Indonesia’s G20 Presidency to promote world recovery, after two years
+          the world has struggled to deal with the Covid-19 pandemic. In this
+          collective recovery process, G20 must be present inclusively, not only
+          paying attention to the interests of its member countries but also the
+          whole world.
+        </p>
+        <h2 class="head-data">2022.10.03(MON) ~ 04(TUE)</h2>
+        <h2 class="main-place">@IdGovernment, Bali, Indonesia</h2>
+      </header>
+
+      <section class="program" id="program-id">
+        <h2 class="program-title">Main Program</h2>
+        <div class="red-line"></div>
+        <div class="main-card">
+          <div class="card">
+            <div class="lecture">
+              <img
```

Comment:
- [ ] Hi, please notice, as mentioned by the previous reviewer, that the text on the making program section cards is still overflowing. 
![image](https://user-images.githubusercontent.com/18730106/148290783-d86f18d2-a0c0-4770-b921-ec54b2cad853.png)

Reply:
## Thanks I did it as you said:
![2022-01-06_06-09-35](https://user-images.githubusercontent.com/63915024/148311950-b8aef09d-2e5b-45b5-9289-0bba78de6c78.png)